.github/workflows/test.yaml

```diff
@@ -138,6 +142,26 @@ jobs:
         key: go-govendor-{{ hashFiles('**/vendor.json') }}
     - name: Get Go dependencies
       run: cd ${{ github.workspace }}/src/github.com/snapcore/snapd && ${{ github.workspace }}/bin/govendor sync
+
+    - name: golangci-lint
+      uses: golangci/golangci-lint-action@v2
+      if: ${{ matrix.gochannel == 'latest/stable' }}
+      with:
+        # version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest`
+        # to use the latest version
+        version: latest
```

Comment:
we probably shouldn't do this and pick a version instead and upgrade once in a while. We don't want to get annoyed/blocked by linter changes